src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,14 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import React from 'react';
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(({ id, ...comment }) => (
+      <CommentInfo
+        key={id}
+        comment={comment}
+      />
+    ))}
```

Comment:
what if you want to use id inside of CommentInfo? not the best idea to use destructuring here, let's just keep it simple, comment.id + comment, fix it everywhere in your code